test/array.specs.ts

```diff
@@ -0,0 +1,81 @@
+import "mocha";
+import fc from "fast-check";
+import { chunk } from "../src/array"; 
+
+describe("chunk: ", () => {
+    it("Should split arrays into chunks of the specified size", () =>
+      fc.assert(
+        fc.property(
+          fc.array(fc.integer(), { minLength: 1 }), //tableau non vide
```

Comment:
Why array of integers ? And why min length ? The case you described above is "any array should be split into chunks" therefore no needs to constraints the type of the array or its length.
Logically, you probably want to test for non-empty arrays, as the behavior for empty one is a bit different, and you probably want to have a number of chunks inferior to your number of element in the  array. You should change the text to show that.
Here is the arbitrary for this case : 
fc.array({minLength: 1}).chain(arr => fc.tuple(fc.constant(arr), fc.integer({max: arr.length })))

Reply:
the test i wanted to made is for non-emtpy arrays thats why the minLength was used ( wasn't clear in my it("...)
So now it made it more clear :  fc.tuple(
          fc.array(fc.anything(), { minLength: 1 }), // Non-empty array
          fc.integer({ min: 1 }) // Size with a minimum value of 1
        ),